holmes/core/supabase_dal.py

```diff
@@ -113,9 +114,81 @@ def get_issue_data(self, issue_id: str) -> Optional[Dict]:
         evidence = (
             self.client
             .table(EVIDENCE_TABLE)
-            .select(f"*")
+            .select("*")
             .filter("issue_id", "eq", issue_id)
             .execute()
         )
-        issue_data["evidence"] = evidence.data
-        return issue_data
\ No newline at end of file
+        enrichment_blacklist = {"text_file", "alert_labels", "ai_analysis", "holmes"}
```

Comment:
Why are you also removing the alert labels? and text files?

Reply:
text files are zipped and require some extra work. also the logs are taken in the investigation so it seems redundant.
alert labels just not sure how useful they are